diff --git a/build/module_args/v8.gni b/build/module_args/v8.gni
index db4ee4e..05b673b 100644
--- a/build/module_args/v8.gni
+++ b/build/module_args/v8.gni
@@ -9,4 +9,7 @@ if (is_android) {
 # TODO(sky): nuke this. Temporary while sorting out http://crbug.com/465456.
 enable_correct_v8_arch = false
 
-v8_use_external_startup_data = !(is_chromeos || is_win)
+# Many V8 users in mojo do not support external startup data for V8, so
+# we disable it. If enabled, special care should be taken to ensure the
+# appropriate startup data is correctly bundled for apps.
+v8_use_external_startup_data = false
