diff --git a/build/config/clang/BUILD.gn b/build/config/clang/BUILD.gn
index 39fe251..721e87c 100644
--- a/build/config/clang/BUILD.gn
+++ b/build/config/clang/BUILD.gn
@@ -21,13 +21,6 @@ config("find_bad_constructs") {
               "//third_party/llvm-build/Release+Asserts/lib/libFindBadConstructs.so",
               root_build_dir) ]
     }
-
-    cflags += [
-      "-Xclang",
-      "-add-plugin",
-      "-Xclang",
-      "find-bad-constructs",
-    ]
   }
 }
 
