diff --git a/net/BUILD.gn b/net/BUILD.gn
index 73eb465..4cecaef 100644
--- a/net/BUILD.gn
+++ b/net/BUILD.gn
@@ -541,6 +541,8 @@ grit("net_resources") {
   ]
 }
 
+if (false) {
+
 static_library("extras") {
   sources = gypi_values.net_extras_sources
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
@@ -550,6 +552,8 @@ static_library("extras") {
   ]
 }
 
+} # if (false)
+
 static_library("http_server") {
   sources = [
     "server/http_connection.cc",
@@ -1124,11 +1128,14 @@ if (!is_android && !is_win && !is_mac) {
   test("net_unittests") {
     sources = gypi_values.net_test_sources
 
+    sources -= [
+      "extras/sqlite/sqlite_channel_id_store_unittest.cc",
+    ]
+
     configs += [ ":net_win_size_truncation" ]
     defines = []
 
     deps = [
-      ":extras",
       ":http_server",
       ":net",
       ":quic_tools",
